packages/devtools_app/lib/src/profiler/cpu_profile_controller.dart

```diff
@@ -212,6 +252,36 @@ class CpuProfilerController with SearchControllerMixin<CpuStackFrame> {
     _processingNotifier.dispose();
     transformer.dispose();
   }
+
+  /// Tracks the identifier for the attempt to filter the data.
+  ///
+  /// We use this to prevent multiple filter calls from processing data at the
+  /// same time.
+  int filterIdentifier = 0;
```

Comment:
make this private? I don't see it used outside of the class.

Reply:
done